lib/fastlane/plugin/revenuecat/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -0,0 +1,110 @@
+require 'fastlane/action'
+require 'fastlane_core/configuration/config_item'
+require_relative '../helper/revenuecat_helper'
+
+module Fastlane
+  module Actions
+    class BumpVersionUpdateChangelogCreatePRAction < Action
+      def self.run(params)
+        # TODO
```

Comment:
Still in progress with this action, will probably be the last one since it's the more complex action.